projects/packages/search/src/dashboard/store/resolvers.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { fetchJetpackSettings, fetchSearchPlanInfo } from './controls';
+import { setJetpackSettings } from './actions/jetpack-settings';
+import { setSearchPlanInfo } from './actions/site-plan';
+import { errorNotice } from '../components/global-notices/store/actions';
+
+/**
+ * Yield actions to get Search Module Status
+ *
+ * @yields {object} - an action object.
+ * @returns {object} - an action object.
+ */
+export function* getSearchModuleStatus() {
+	try {
+		const settings = yield fetchJetpackSettings();
+		if ( settings ) {
+			return setJetpackSettings( settings );
+		}
+	} catch ( e ) {
+		return errorNotice( __( 'Error fetching settings…' ) );
```

Comment:
```suggestion
		return errorNotice( __( 'Error fetching settings…', 'jetpack' ) );
```

Reply:
@jeherve oops, thanks for catching this, will address in the following PR.